src/app/app.routes.ts

```diff
@@ -15,8 +15,9 @@ export const ROUTES: Routes = [
   },
   {
     path: 'caseworker-details',
-    canActivate: [AuthGuard],
-    loadChildren: '../case-worker-ref-data/case-worker-ref-data.module#CaseWorkerRefDataModule'
+    canActivate: [AuthGuard, RoleGuard],
+    loadChildren: '../case-worker-ref-data/case-worker-ref-data.module#CaseWorkerRefDataModule',
+    data: {needsRole: ['cwd-admin'], roleMatching: 0 }
```

Comment:
Should this be `roleMatching: RoleMatching.ALL`?

Reply:
Thats the change I made. When you use enum in Guard the PROD build fails

Reply:
This is fixed now